src/library.h

```diff
@@ -61,4 +61,6 @@ public slots:
 friend class LibraryManipulator;
 };
 
+zim::Entry getArchiveEntryFromUrl(const zim::Archive& archive, const QUrl& url);
+
```

Comment:
Declaring functions in header files is convenient but not mandatory. You can simply forward declare it in the second .cpp file (accompanied by a comment pointing to the location of the definition) and do that in the commit when that second usage is introduced. This way the issue of finding an appropriate header file for this declaration (and the ensuing sense of necessity of moving the definition in the respective cpp file) is worked around.

Reply:
Done